jsonifier/models/models.py

```diff
@@ -83,6 +83,11 @@ def _jsonify_record(self, parser, rec, root):
             except SwallableException:
                 continue
             json_key = field_dict.get("target", field_dict["name"])
+            if rec.env.context.get("default_function") and not field_dict.get(
+                "function"
+            ):
+                # after all processing we post-process the parser to impose a function.
+                field_dict["function"] = "default_function"
```

Comment:
If you do it like this, then, if the `field_dict` already has a function, this won't work. That would be a limitation. You could overcome that by moving this to the end of the function, where `value` is already guaranteed to be defined and where you can get `field` by doing `field = rec._fields[field_dict["name"]]`. Then, you can do:

```
if self.env.context.get("default_function"):
    value = {'value': self.default_function(field)}
```

Reply:
Very nice, but it was wrong of me to piggyback on the function feature, wich sould remain untouched.
Plus it did not allow me to add a "sister" key, just change the value.  Alnmost done with a simple  version.
Works. Still in WIP

Reply:
function solution was trashed. tested and works with a more simple solution.